cw-relayer/cmd/price-feeder.go

```diff
@@ -0,0 +1,186 @@
+package cmd
```

Comment:
Let's rename this file to `cw-relayer.go`

That way we don't conflating the two